backend/src/main/scala/cromwell/backend/async/AsyncBackendJobExecutionActor.scala

```diff
@@ -72,8 +72,8 @@ trait AsyncBackendJobExecutionActor { this: Actor with ActorLogging =>
     case PollResponseReceived(handle) if handle.isDone => self ! Finish(handle)
     case PollResponseReceived(handle) =>
       context.system.scheduler.scheduleOnce(pollBackOff.backoffMillis.millis, self, IssuePollRequest(handle))
-    case Finish(SuccessfulExecutionHandle(outputs, returnCode, resultsClonedFrom)) =>
-      completionPromise.success(SucceededResponse(jobDescriptor.key, Some(returnCode), outputs))
+    case Finish(SuccessfulExecutionHandle(outputs, returnCode, callOutputFiles, resultsClonedFrom)) =>
+      completionPromise.success(SucceededResponse(jobDescriptor.key, Some(returnCode), outputs, Some(callOutputFiles)))
```

Comment:
For the sake of consistency, I'll rename callOutputFiles or jobOutputFiles --> jobDetritusFiles. Alteast when we rename JobDetritus, it'll be very clear what needs to be altered.